xyzservices/_parse_leaflet_providers.py

```diff
@@ -56,7 +56,7 @@ def get_json_data():
 
```

Comment:
This file doesn't actually need to live in the package? (in contextily I had put it in `scripts/`, but could also live in the `provider_sources/` top-level dir)

Reply:
It doesn't. I moved both files to the `provider_sources` dir.